docs/jobs/ai-assistant.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Using OpenFn with AI Assistant
+sidebar_label: Using AI Assistant
+---
+
+:::warning
+
+AI makes mistakes. Always review the AI's response carefully and make sure to
+verify the output. You can help us improve the AI Assistant by providing
+feedback on the response.
+
+:::
+
+## About OpenFn AI Assistant
+
+The AI Assistant is built on the Claude 3.5 Sonnet model from Anthropic and is
+trained with OpenFn documentation and example job code. The goal of the
+assistant is to help you with your jobs and diagnosing issues with your jobs and
+error logs. It is important to note that the assistant is not a replacement for
+human developers. It is a tool to help you with your jobs and diagnosing issues
+with your jobs and error logs.
+
+:::caution
+
+Before you start, we recommend that you take a moment to read our more about the
```

Comment:
First of all it's the Responsible AI Policy. Please use that branding when you refer to it.

Secondly, while I want the policy to be available, accessible, and occasionally actually read by users - I would never suggest that users read it "first".

A better approach to this would be a section called "Using AI Responsible" (which might overlap "how accurate is it"). That would include a super brief summary of the policy (like a sentence) and a link to it.

Thirdly, I don't like that this is in a scary caution block. I don't want to scream DANGER DANGER about this stuff. None of this is designed to scare users away or put them off. I want them using our model, not chatgpt, for questions. The responsible AI stuff is just about softly making users think critically when using it, which will actually result in a better UX anyway.